flocker/restapi/docs/test/test_publicapi.py

```diff
@@ -123,7 +123,7 @@ def hasExamples():
                 u"response":
                     u"HTTP/1.1 200 OK\n"
                     u"\n"
-                    u'{"error": false, "result": "%(DOMAIN)s"}\n',
+                    u'"%(DOMAIN)s"\n',
```

Comment:
Maybe stick some other arbitrary structure around this value now.  A bare string isn't legal JSON, I think.